requirements/pytorch/test.txt

```diff
@@ -14,3 +14,4 @@ psutil<5.9.4 # for `DeviceStatsMonitor`
 pandas>1.0, <1.5.2  # needed in benchmarks
 fastapi<0.87.0
 uvicorn<0.19.1
+tensorboard>=2.9.1, <2.11.0
```

Comment:
Do we know what happens when both are installed? `tensorboardX` is still used because of the `tensorboardX` import, no?

I think we should do this

```python
if _TENSORBOARD_AVAILABLE:
    from tensorboard ...
elif _TENSORBOARDX_AVAILABLE:
    from tensorboardX ...
```

Reply:
ok let's do this

Reply:
I do not think so, as `tensorboardx` is importing `tensorboard` in if you request logging graph

Reply:
Right, but if `tensorboard` is present then we can use it fully (not just for graph).

Reply:
What fully you mean? Logger here is just interface to dump your data, for visualization you can use still both but independent on PL...